ykrt/src/compile/jitc_yk/jit_ir/mod.rs

```diff
@@ -2158,30 +2160,32 @@ impl LoadInst {
 ///
 /// Specifies the [yksmp::Location] of a run-time argument.
 #[derive(Clone, Copy, Debug)]
-#[repr(packed)]
 pub struct ParamInst {
-    /// The [yksmp::Location] of this input.
-    locidx: u32,
+    /// The [InstIdx] of the [yksmp::Location] parameter.
+    paramidx: InstIdx,
```

Comment:
I've gotten confused by this. Are you are reusing `InstIdx` for indices of a vector other than `self.insts`? If that's what's happening, shouldn't we introduce a `ParamIdx`?

Reply:
I see. It's a little confusing still. Up to you guys.

Reply:
Basically (and I think this is sensible!) parameters come first, contiguously. `ParamIdx` slightly obfuscates this. Could we do better? Maybe. But I think that's for a future PR.